.gitignore

```diff
@@ -40,3 +38,12 @@ storybook-static/
 
 # lighthouse
 localhost_*.report.html
+
+# yarn
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

Comment:
このへんのファイルはignoreしないほうが良いのか。どういう役割・理由でignoreしてるのか気になった

Reply:
依存パッケージを.yarnディレクトリ配下にキャッシュするらしいんで、それらをignoreしている感じっす！
https://zenn.dev/wakamsha/articles/migrate-from-npm-to-yarn#4.-.gitignore-%E3%81%AB-yarn-%E7%94%A8%E3%81%AE%E8%A8%AD%E5%AE%9A%E3%82%92%E8%BF%BD%E8%A8%98%E3%81%99%E3%82%8B

Reply:
むしろ

```
!.yarn/patches
!.yarn/plugins
!.yarn/releases
!.yarn/sdks
!.yarn/versions
```

このへんのファイルをなんでignoreしてないんだろう(厳密にはignoreを打ち消してる)と思った感じかな

Reply:
勘違いしてた...失礼...🙏
公式的にはignoreしないファイルは依存関係をインストールする関係上、必要になるらしいっす（↓は `.yarn/patches` の説明です）

> .yarn/patches contain the patchfiles you've been generating with the [yarn ](https://yarnpkg.com/cli/patch-commit)[patch-commit](https://yarnpkg.com/cli/patch-commit) command. You always want them in your repository, since they are necessary to install your dependencies.
>
> 訳）.yarn/patchesには、yarn patch-commitコマンドで生成したパッチファイルが含まれています。これらは依存関係をインストールするのに必要なので、常にリポジトリに置いておきたいものです。

https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored